consensus/istanbul/core/types.go

```diff
@@ -27,6 +27,8 @@ type Engine interface {
 	Stop() error
 	CurrentView() *istanbul.View
 	SetAddress(common.Address)
+	// Validator -> CommittedSeal from Parent Block
+	ParentCommits() *messageSet
```

Comment:
Can this function be removed from the interface?   The functions listed here are normally for packages that are outside of the top level `consensus` package.

Reply:
The goal here is to access the data inside `<core>.current.ParentCommits` of `core` from `backend.go`. [`core` is a field of `Backend`](https://github.com/celo-org/celo-blockchain/blob/eb9ad80d99ddc43e48a3d133e359e4c98385ce3c/consensus/istanbul/backend/backend.go#L99) which implements the Engine interface. 

I did not find any mentions of `CurrentView` and `SetAddress` outside of `consensus/istanbul/core` (ie outside of the top level package), so it made sense to expose the desired data over the interface and add an extra function in the implementation of the interface under the `core/handler.go` file (where the rest of the interface is also implemented).

In order to access the core as a `core` type, we'd have to export the core (as `Core`) in order to do something like `sb.core.(*Core)`, and then get the data, but it doesn't make much sense to me to do that.

Does that convince you to go with the interface approach as I already have, or do you prefer the exporting option? Do you have any alternative suggestions?

Reply:
You approach makes sense to me.